### PR TITLE
Added links to PowerShell cmdlets

### PR DESCRIPTION
**Pull request by ChrisKibble**
Added links to how to accomplish the manual tasks in an automated way using PowerShell.

#MMS2019Docathon submission

### CODE REVIEW

**Comment by PRMerger12** [comment]
@ChrisKibble : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @ChrisKibble for the contribution! I'll review after the end of the contest.

#hold-off
(reviewers, please don't edit - I'll take care of it.)

File: sccm/core/clients/manage/collections/create-collections.md [hunk 1]
```diff
@@ -213,3 +213,10 @@ For more information about exporting collections, see [How to manage collections
 
 5. Complete the wizard to import the collection. The new collection is displayed in the **User Collections** or **Device Collections** node of the **Assets and Compliance** workspace. Refresh or reload the Configuration Manager console to see the collection members for the newly imported collection.  
 
+## <a name="bkmk_powershell"></a> Using PowerShell
+
+PowerShell can be used to create and import collections.  For more information, see:
+
+* [New-CMCollection](https://docs.microsoft.com/en-us/powershell/module/configurationmanager/new-cmcollection)
```
Comment (aczechowski) [inline sccm/core/clients/manage/collections/create-collections.md]: At a minimum, drop the locale "en-us/" from the URLs

Technically, they should all be lowercase, but I don't think that's critical right now.
Also, I'm honestly not sure what the "?view=sccm-ps" parameter is at the end of live URLs. I usually include them, but right now it does work without them.

**Comment by PRMerger14** [comment]
@ChrisKibble : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

File: sccm/core/clients/manage/collections/create-collections.md (hunk 1 above)
Comment (ChrisKibble) [reply sccm/core/clients/manage/collections/create-collections.md]: Thanks @aczechowski, when I go to the URLs it appends that to the URL, but it didn't appear to be required and I didn't see it referenced in any of the links, so I excluded it.  I've resolved the locale issue and will keep this in mind on future PRs.

**Comment by aczechowski** [comment]
#sign-off 
*Once merged*, this should be live with the next daily publishing process. Right now that *should* be tomorrow (Wednesday, 15 May) around 11:00 Pacific time. No further action needed from you.

**Pull request merged**